include/aspect/particle/property/composition_reaction.h

```diff
@@ -78,6 +78,17 @@ namespace aspect
           InitializationModeForLateParticles
           late_initialization_mode () const override;
 
+          /**
+           * A function that returns which advection field to be used
```

Comment:
```suggestion
           * A function that returns the advection field to be used
```
Or which advection field **is** to be used

Reply:
And the same for all the places below.